tests/TaxonomyTermTest.php

```diff
@@ -43,4 +43,22 @@ public function testSorting() {
 		$this->assertEquals($fruit->ID, $plant->Children()->first()->ID);
 		$this->assertEquals($vegetable->ID, $plant->Children()->last()->ID);
 	}
+	
+	/**
+	 * Tests correct arguments, get and create a taxonomy
+	 */
+	public function testGetOrCreate() {
```

Comment:
This test seems a bit rudimentary. You should try to get a term that you know exists and assert that it's ID matches the ID of the existing term.